### DISCUSSION TITLE
Fermo per manutenzione straordinaria

### DISCUSSION THREAD

**Discussion by pdonadeo** [pdonadeo/e-uscito-joypad#33] [Announcements]
Vi scrivo per avvisarvi che nel fine settimana ci sarà un fermo dei siti (è uscito Joypad, lo staging ed anche il database) perché il server che li ospita è in fase di dismissione e quindi sposterò tutto altrove.

Non so ancora quando di preciso ma vi avviserò con un paio d'ore di spegnere tutto.

[Trenord si scusa](https://nonciclopedia.org/wiki/Trenord).

**Comment by pdonadeo** [comment]
È uscitə l'epipuntatə 52: anticipo la manutenzione perché poi va aggiunto il nuovo materiale al database.

Quindi: tutto "down" per qualche ora.

**Comment by pdonadeo** [comment]
Manutenzione conclusa, è tutto on line. Ho anche aggiunto la scheda per l'episodio che è appena uscito. Però ora me lo ascolto che va bene tutto ma si parla di Monkey Island 😃

**Reply by MarcoJul** [reply]
Direi che non potrei essere più d'accordo di così😅
